.gitlab-ci.yml

```diff
@@ -203,23 +206,38 @@ release-docker-hub-tag:
   variables:
     TAG: "${CI_COMMIT_TAG}"
 
+# notify slack when the pre-release job fails
 slack-notifier-pre-release.on-failure:
   extends: .slack-notifier-base
   variables:
     CI_PIPELINE_NAME: "Pre-Release"
```

Comment:
Can't we use the predefined variable `CI_JOB_STAGE` instead of setting the pipeline name here?

Reply:
CI_JOB_STAGE would be equivalent to notify because the notifications happen at the notify stage, so it wouldnt work as intended.